apps/meteor/tests/end-to-end/api/09-rooms.js

```diff
@@ -1707,4 +1707,280 @@ describe('[Rooms]', function () {
 				.end(done);
 		});
 	});
+	describe('/rooms.isMember', async () => {
+		let testChannel;
+		let testGroup;
+		let testDM;
+
+		const fakeRoomId = `room.test.${Date.now()}-${Math.random()}`;
+		const fakeUserId = `user.test.${Date.now()}-${Math.random()}`;
+
+		const testChannelName = `channel.test.${Date.now()}-${Math.random()}`;
+		const testGroupName = `group.test.${Date.now()}-${Math.random()}`;
+
+		let testUser1;
+		let testUser2;
+		let testUserNonMember;
+		let testUser1Credentials;
+		let testUserNonMemberCredentials;
+
+		before(async () => {
+			[testUser1, testUser2, testUserNonMember] = await Promise.all([createUser(), createUser(), createUser()]);
```

Comment:
I'm still not so sure of why these tests are failing (and they're also flaky)
Maybe try not using `Promise.all` here specifically. I'm wondering if maybe their usernames are coliding due to the timestamp (since createUser uses `Date.now()` to define each username)

Reply:
Yes, It might be due to Date.now() in usernames as I was using promise.all for creating users. I have updated the code to create user sequentially